tests/test_pipeline.yml

```diff
@@ -0,0 +1,14 @@
+- name: test_dry_run
+  command: snakemake  -n -r -p -s Snakefile --config GNOMAD_DATA_DIR=tests/data/gnomad_data/ REFERENCE_FASTA=tests/data/ref.fa
+- name: test_config_errors
+  command: snakemake  -n -r -p -s Snakefile
```

Comment:
I would add `REFERENCE_FASTA=...` to this line as well so you can be sure it tests for `GNOMAD_DATA_DIR` only.